tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/settings_form.py

```diff
@@ -35,18 +36,13 @@ def __init__(self, marionette):
         Base.__init__(self, marionette)
         view = self.marionette.find_element(*self._settings_view_locator)
         Wait(self.marionette).until(lambda m: view.location['y'] == 0)
+        self._order_by_last_name_switch = GaiaBinaryControl(self.marionette, self._order_by_last_name_switch_locator)
 
-    def tap_order_by_last_name(self):
-        last_name = Wait(self.marionette).until(
-            expected.element_present(*self._order_by_last_name_switch_locator))
-        Wait(self.marionette).until(expected.element_displayed(last_name))
-        initial_state = self.is_custom_element_checked(last_name)
-        last_name.tap()
-        self.wait_for_custom_element_checked_state(last_name, checked=not(initial_state))
+    def enable_order_by_last_name(self):
```

Comment:
Perhaps we could just do remove this method and just use contacts_settings.order_by_last_name_switch.enable() directly in the test?
This just feels like an unnecessary inbetween step to me.